ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -437,5 +448,11 @@ class MSFNotificationStateImpl: NSObject, ControlConfiguration, MSFNotificationS
         self.actionButtonTitle = actionButtonTitle
         self.actionButtonAction = actionButtonAction
         self.messageButtonAction = messageButtonAction
+        self.showFromBottom = showFromBottom
+        if let showDefaultDismissActionButton = showDefaultDismissActionButton {
```

Comment:
i don't have swift compiler in my head but self.showDefaultDismisseActionButton = showDefaultDismissActionButton ?? style.isToast dosn't work?

Reply:
Ah, yeah, that should also work.